pointgrey_camera_driver/CMakeLists.txt

```diff
@@ -94,6 +94,22 @@ install(FILES nodelet_plugins.xml DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION
 
 install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
 
+if(CATKIN_BUILD_BINARY_PACKAGE)
+  # Install udev rules with dh_installudev: http://manpages.ubuntu.com/manpages/man1/dh_installudev.1.html
+  STRING(REPLACE "_" "-" DEBIAN_PACKAGE_NAME ros-$ENV{ROS_DISTRO}-${PROJECT_NAME})
+  FILE(REMOVE ${CMAKE_CURRENT_SOURCE_DIR}/debian/${DEBIAN_PACKAGE_NAME}.udev)
+  FILE(GLOB UDEV_FILES RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}/udev" "${CMAKE_CURRENT_SOURCE_DIR}/udev/*.rules")
+  FOREACH(FILENAME ${UDEV_FILES})
+    MESSAGE(STATUS "Appending contents of udev/${FILENAME} to debian/${DEBIAN_PACKAGE_NAME}.udev")
+    FILE(READ udev/${FILENAME} FILEDATA)
+    FILE(APPEND ${CMAKE_CURRENT_SOURCE_DIR}/debian/${DEBIAN_PACKAGE_NAME}.udev ${FILEDATA})
+  ENDFOREACH(FILENAME)
```

Comment:
I think in principle, I'd prefer to see this done as an `add_custom_target` + `add_custom_command`, where as script generates the file at build time, rather than doing it all at configure time. But the logic is small enough that I suppose it's fine as is.

The change I _would_ like to see made, though, would be using lowercase CMake commands, to match the prevailing style in this file.